app/pages/patients/patients.js

```diff
@@ -98,7 +94,7 @@ export let Patients = React.createClass({
     var handleClickYes = function(e) {
       e.preventDefault();
       self.props.onHideWelcomeSetup();
```

Comment:
If you get distracted and close the tab, nothing happens anyway -> you have not clicked yes.